src/app/shared/entity/provider/eks.ts

```diff
@@ -29,6 +29,7 @@ export class EKSClusterSpec {
   roleArn: string;
   version?: string;
   vpcConfigRequest: {
+    vpcId: string;
```

Comment:
This will cause issue when Types forms are used currentl we are using in Create Cluster wizard

like `as EKSClusterSpec` but it is not throwing any warning but still it should be `optional` property instead of required

```suggestion
    vpcId?: string;
```